src/auth/userSession.ts

```diff
@@ -31,13 +33,16 @@ import { BLOCKSTACK_DEFAULT_GAIA_HUB_URL, DEFAULT_BLOCKSTACK_HOST, DEFAULT_SCOPE
  *
  * A user can be signed in either directly through the interactive
  * sign in process or by directly providing the app private key.
+ * 
+ * @type {UserSession}
+ * 
  */
 export class UserSession {
```

Comment:
I noticed that links to type from external files are sometimes not generated. For example `GetFileOptions` here:

----

![image](https://user-images.githubusercontent.com/1447546/55505834-fe305280-5621-11e9-8e35-632d7b26ce43.png)

----

I found that using typescript's [`import() types syntax`](https://davidea.st/articles/typescript-2-9-import-types) fixes the issue. Example with `getFile(path: string, options?: import('../storage').GetFileOptions)`:

----

![image](https://user-images.githubusercontent.com/1447546/55506157-c07ff980-5622-11e9-8968-89d1d681a925.png)

Reply:
I'd like to fix this after the merge. I have a number of other things to do. This is just the first pass.

Reply:
This worked for the interface `GetFileOptions` but does not work for the type `PutFileOptions`.